### PR TITLE
[DO NOT MERGE] devnet testing for FIP-0079

### PR DESCRIPTION
**Pull request by hanabi1224**
## Summary of changes

This PR runs [`Integration tests / Devnet checks`](https://github.com/ChainSafe/forest/actions/runs/9077302920/job/24942095249?pr=4321) with a custom lotus image `LOTUS_IMAGE=ghcr.io/chainsafe/lotus-devnet:2024-05-14-0ac3cf4` that is [built](https://github.com/ChainSafe/forest/actions/runs/9077091800/job/24941194618#step:3:62) from the `FIP0079-testing` branch in `lotus`.

To run the devnet checks locally:
1. Install `rust` and `docker` toolchains
2. Checkout `hm/devnet-test-FIP-0079` branch of `forest`
3. Run `./scripts/devnet/setup.sh`
4. Run `./scripts/devnet/check.sh`
5. Run `docker logs -f [forest|lotus]` to inspect logs

## Reference issue to close (if applicable)

Closes

## Change checklist

- [ ] I have performed a self-review of my own code,
- [ ] I have made corresponding changes to the documentation,
- [ ] I have added tests that prove my fix is effective or that my feature works (if possible),
- [ ] I have made sure the [CHANGELOG][1] is up-to-date. All user-facing changes should be reflected in this document.

[1]: https://github.com/ChainSafe/forest/blob/main/CHANGELOG.md

### CODE REVIEW

File: scripts/devnet/lotus.dockerfile [hunk 1]
```diff
@@ -13,6 +13,7 @@ RUN git clone --depth 1 --branch FIP0079-testing https://github.com/filecoin-pro
 
 RUN CGO_CFLAGS_ALLOW="-D__BLST_PORTABLE__" \
     CGO_CFLAGS="-D__BLST_PORTABLE__" \
+    FFI_USE_BLST_PORTABLE="1" \
```
Comment (LesnyRumcajs) [inline scripts/devnet/lotus.dockerfile:16]: Should we have a mergeable PR with this?

Comment (hanabi1224) [reply scripts/devnet/lotus.dockerfile:16]: Will do

Comment (hanabi1224) [reply scripts/devnet/lotus.dockerfile:16]: https://github.com/ChainSafe/forest/pull/4327

**Comment by LesnyRumcajs** [comment]
@hanabi1224 shall we close this?

**Comment by hanabi1224** [comment]
@LesnyRumcajs Yes! Done.

**Pull request closed without merging**